tools/keycloak/sync/realm.json

```diff
@@ -127,7 +127,7 @@
         "AuthorizationServicesEnabled": false,
         "RedirectUris": [
           "https://pims-app-3cd915-prod.apps.silver.devops.gov.bc.ca/api/*",
-          "https://pims.th.gov.bc.ca/api/*"
+          "https:/pims.th.gov.bc.ca/api/*"
```

Comment:
what is going on here?

Reply:
Bad URI

Reply:
fixed

Reply:
@sookeke this line (130) still remains unchanged. Please fix like the others. thanks!